SampleComponentLibraryApp/SampleComponentLibraryApp/Theme+Extensions.swift

```diff
@@ -0,0 +1,20 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+
+let defaultSampleComponentUUID = UUID(uuidString: "44BA0B7D-097A-484D-8358-91A6E374451D")!
```

Comment:
copied over from client side, but happy to use `UUID()` if that's preferred

Reply:
I don't think in this case the specific UUID value should matter so this should be fine. 👍 That UUID was intended originally for the UI tests but we can use it here, I'm not aware of any conflict it would cause.